rails/app/controllers/dashboard/users_controller.rb

```diff
@@ -34,11 +34,11 @@ def create
     end
 
     def show
-      @user = authorize community.users.find(params[:id])
+      @user = authorize User.find(params[:id])
```

Comment:
one thing that doing the find off a `Community`'s `users` association is that it was implicitly a permissions check too, because `community` is getting pulled from the `current_user` here https://github.com/Terrastories/terrastories/blob/62443134bac8662c8bc2df76e36596b7f81453f0/rails/app/controllers/dashboard/application_controller.rb#L27-L30

If you remember from our discussion, the reason we have to remove it here is that a super admin doesn't belong to a single community, so if we remove this implicit community check, we'd probably have to add an explicit one in `UserPolicy`

Reply:
Oh sorry, I wasn't clear what the consequence of this is. I believe without this implicit `community` check, an `admin` of any `community` can access these routes for **other** communities.

Reply:
Yep, you're right! Added logic and tests to confirm the admin is within the same community.